tests/api.c

```diff
@@ -83658,6 +83658,13 @@ static int test_CryptoCb_Func(int thisDevId, wc_CryptoInfo* info, void* ctx)
                 info->pk.rsa.type, ret, *info->pk.rsa.outLen);
         #endif
         }
+        #ifdef WOLF_CRYPTO_CB_RSA_PAD
+        else if (info->pk.type == WC_PK_TYPE_RSA_PKCS ||
+                 info->pk.type == WC_PK_TYPE_RSA_PSS  ||
+                 info->pk.type == WC_PK_TYPE_RSA_OAEP) {
+            ret = CRYPTOCB_UNAVAILABLE; /* fallback to software */
```

Comment:
I would like to see a software implementation of this.
We can expose APIs in rsa.c to help make this work.

Reply:
I implemented a software leg for the RSA private key operations similar to how the existing callback for testing operates. I was able to do it with existing APIs, let me know if that is what you had in mind.